packages/one-app-bundler/bin/webpack-bundle-module.js

```diff
@@ -12,25 +12,28 @@
  * under the License.
  */
 
-const webpack = require('webpack');
-const path = require('node:path');
-const fs = require('node:fs');
-const localeBundler = require('@americanexpress/one-app-locale-bundler');
+import webpack from 'webpack';
+import path from 'node:path';
+import fs from 'node:fs';
+import localeBundler from '@americanexpress/one-app-locale-bundler';
+import getConfigOptions from '../utils/getConfigOptions.js';
+import getWebpackCallback from './webpackCallback.js';
+import utils from '../utils/getCliOptions.js';
+import clientConfig from '../webpack/module/webpack.client.js';
+import serverConfig from '../webpack/module/webpack.server.js';
 
-const getConfigOptions = require('../utils/getConfigOptions');
-const clientConfig = require('../webpack/module/webpack.client');
-const serverConfig = require('../webpack/module/webpack.server');
-const getWebpackCallback = require('./webpackCallback');
-const { watch } = require('../utils/getCliOptions')();
+export const webpackBundleModule = async () => {
+  const { watch } = utils();
 
-const modernClientConfig = clientConfig('modern');
-const legacyClientConfig = clientConfig('legacy');
+  const modernClientConfig = await clientConfig('modern');
+  const legacyClientConfig = await clientConfig('legacy');
```

Comment:
promise.all?

Reply:
Yah good idea, that said its ultimately awaiting a dynamic import, so its a 'fast' promise

Reply:
No thats fine, its a small optimization that isn't required